firedantic/models.py

```diff
@@ -49,10 +63,24 @@ def find(cls: Type[TModel], filter_: dict) -> List[TModel]:
         query = coll
```

Comment:
A few rows up there's still a comment saying `Currently only supports `==` operator.`; could you please remove that?

Reply:
Maybe also add another example with the operators there at the same time.

Reply:
Done